packages/cadl-ranch-specs/http/parameters/collection-format/mockapi.ts

```diff
@@ -7,7 +7,7 @@ Scenarios.Parameters_CollectionFormat_Query_multi = passOnSuccess(
   mockapi.get("/parameters/collection-format/query/multi", (req) => {
     req.expect.containsQueryParam("colors", ["blue", "red", "green"], "multi");
     return {
-      status: 200,
+      status: 204,
```

Comment:
could we verify that the mockapis return the correct status code? i.e. 204 if they're returning NoContentResponse? This is more of a stretch ask, there might be some difficulties in adding this that i'm not anticipating though. thanks!

Reply:
we could maybe but thats probably a lot of work so not sure its worth it